### ISSUE TITLE
Add support for web

### ISSUE THREAD

**Issue by mattapperson** [eridem/patata#13]
should not be too hard... then we have one tool to rule them all!

**Comment by eridem** [comment]
Agree. Actually, I was thinking about doing this nowadays too. `patata` should be smart enough to run it for web without change a single line of code for the developers.
